src/graph_world/nodeclassification/generator_wrappers.py

```diff
@@ -53,14 +56,36 @@ def Generate(self, sample_id):
         self._marginal)
     generator_config['generator_name'] = 'StochasticBlockModel'
 
+    # Generating LFR graph to copy community sizes and compute pi and prop_mat
+    if self._use_generated_lfr_communities: 
+      _, lfr_model = SimulateLFRWrapper(generator_config['nvertex'], 
+                                        self._lfr_params['avg_degree'],
+                                        int(self._lfr_params['max_degree_proportion']*generator_config['nvertex']),
+                                        self._lfr_params['power_exponent'],
+                                        int(self._lfr_params['community_min_size_proportion']*generator_config['nvertex']), 
```

Comment:
As mentioned in my other comment, it would be nice to add samplers here for the LFR parameters that matter to the community size generation. That would allow the user to fully use LFR community generation as a surrogate for SBM community generation.